src/app/zone-egresses/views/IndexView.vue

```diff
@@ -8,23 +8,24 @@
       v-slot="{ route, t }"
       name="zone-egress-list-view"
       :params="{
-        page: 1,
-        size: me.pageSize
+        /* page: 1, */
+        /* size: me.pageSize, */
```

Comment:
Once we have backend filtering we'll put all this back, for now we just hardcode `page=1` and `size=100` in the URI for the DataSource.